Issue #344 mobile design

VWRoli opened the pull request:
# Description

Font sizes are responsive on their own now (check the design first though, you might find some differences from the main style from time to time).
Padding and margins are not, they have to be done manually. Did this on the planter and tree page.

![Screenshot from 2021-12-13 11-22-55](https://user-images.githubusercontent.com/31432331/145798203-45261a57-dc75-4966-be52-d752dddcba55.png)![Screenshot from 2021-12-13 11-23-09](https://user-images.githubusercontent.com/31432331/145798214-bc755937-2603-4a66-bb97-6a2135e54edb.png)
![Screenshot from 2021-12-13 11-23-13](https://user-images.githubusercontent.com/31432331/145798227-a4efa7fc-4e5b-4337-9c31-349a83bf2515.png)



Fixes #344

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Cypress component tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

QuaidBartolomei reviewed (changes requested):
There are some other linting and formatting issues that can be automatically fixed with the `lint:fix` and `format` scripts. If you have `lint-staged` setup formatting and linting will happen automatically when you use `git commit`. To make sure `lint-staged` is setup you can run `npm i`.

VWRoli commented:
> There are some other linting and formatting issues that can be automatically fixed with the `lint:fix` and `format` scripts. If `lint-staged` is working properly then this formatting and linting will happen automatically when using `git commit`. To make sure `lint-staged` is setup you can run `npm i`.

Fixed the linting errors although lint-staged is running when I make a commit, don't know what could be the issue.
![Screenshot from 2021-12-15 16-09-38](https://user-images.githubusercontent.com/31432331/146212598-fcfed6bf-42f3-4073-84b3-095aaf60adf8.png)

Also renamed the `colorTheme`.

QuaidBartolomei commented:
@VWRoli You are right, the lint-staged settings need to be changed. Thanks for checking on it 👍.
